ctrlf.el

```diff
@@ -361,35 +370,38 @@ as a backport for Emacs 26 and earlier where signaling a message
 while the minibuffer is active causes an absolutely horrendous
 mess."
   ;; Some of this is borrowed from `minibuffer-message'.
-  (let ((string (apply #'format (concat " [" format "]") args)))
-    (put-text-property 0 (length string) 'face 'minibuffer-prompt string)
```

Comment:
This looks like way more change than it actually is because of indentation. I just added handling of `ctrlf--minibuffer` being `nil` (and ensured it is such if a ctrlf search isn't active) such that the message is displayed consistently as if `minibuffer-message` were used directly. This lets us use `ctrlf--message` even if a search _wasn't_ started (like no symbol at point was found).

Reply:
(Try `M-s .` with no symbol at point, then cancel and try `C-s` followed by `M-s .` for a demo of the logic.)

Reply:
Awesome.